htdocs/scss/skins/lynx.scss

```diff
@@ -1,40 +1,64 @@
-// color names. Only use these variables in the color assignment section
-$_black:     #000;
-$_white:     #FFF;
-$_dark-gray: #636363;
+// The Lynx skin deliberately does not look or act like a proper site skin. It
+// should defer to browser default styles wherever possible.
 
-$_support-bg-blue:  #c5dff9;
-$_support-fg-blue:  darken( $_support-bg-blue, 15% );
-$_support-bg-green: #d0fbca;
-$_support-fg-green: darken( $_support-bg-green, 15% );
+* {
+  box-sizing: border-box;
+}
 
-// color assignment
-$text-color: $_black;
-$primary-text-color: $_white;
-$inactive-color: $_dark-gray;
-$background-color: $_white;
-$soft-accent-color: $_white;
+@import "foundation/foundation_minimal";
+// ^^ used for pages that need foundation features but should never LOOK like
+// foundation.
 
-@import "skins/alert-colors";
+// Color variables required by compatibility-styles:
+$highlight-color: unset;
+$highlight-text-color: unset;
+$highlight-border: unset;
+@import "skins/compatibility-styles";
 
-$body-bg:           $background-color;
-$body-font-color:   $text-color;
-$form-font-color:   rgba(0,0,0,0.75);
-$form-bg-color:     #f7f7f7;
+// Color variables required by entry-styles:
+$soft-accent-color: unset;
+$secondary-color-alternate: #c0c0c0;
+$secondary-color: #e2e2e2;
+$screened-comment-bg: #C29B9B; // old Tropo screened comment color, why not.
```

Comment:
Technically old Lynx had its own screened comment background color, but nobody's ever seen it because the CSS specificity was messed up and it got overridden by the default odd/even backgrounds. But if you go throw an !important on there in the web inspector, you'll see that it's unacceptable anyway.